v3/README.md

```diff
@@ -87,3 +104,53 @@ GENERATOR_DEBUG=client make generate > test/client.go
 GENERATOR_DEBUG=schemas make generate > test/schemas.go
 GENERATOR_DEBUG=operations make generate > test/operations.go
 ```
+
+### OpenAPI Extensions
+
+The generator support two types of extension:
+- `x-go-type` to specify a type definition in Golang.
+
+	OpenAPI Spec
+	```yaml
+	api-endpoint:
+	  type: string
+	  x-go-type: Endpoint
+	  description: Zone API endpoint
+	```
+	Generated code
+	```Golang
+	type Endpoint string
+
+	type Zone struct {
+		APIEndpoint Endpoint // Here is the generated type definition.
+		...
+	}
+	```
+- `x-go-findable` to specify which fields in the findable resource to fetch
```

Comment:
From my understanding of the code, there are some limitations worth documenting:
- `x-go-findable` can only be set on string types;
- max 2 attributes can have a `x-go-findable`;
- values of `x-go-findable` can be either `"1"` or `"2"` , where former value determines string will be search there first.